db/migrate/20170605003515_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[5.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+      t.string :author
+      t.string :date
```

Comment:
Consider using the `.datetime` datatype so you have access to Ruby's datetime methods.